File: scrap.py
import requests
from bs4 import BeautifulSoup
from csv import writer
response = requests.get('https://www.sample.com/about.html')
soup = BeautifulSoup(response.text, 'html.parser')
posts = soup.find_all(class_='box-info')
with open('posts.csv', 'w') as csv_file:
    csv_writer = writer(csv_file)
    headers = ['Title', 'Link', 'Date']
    csv_writer.writerow(headers)
    for post in posts:
        title = u''.join(post.find(class_='title').get_text().replace('\n', '')).encode('utf-8').strip()
        link = u''.join(post.find('p').get_text().replace('\n', '')).encode('utf-8').strip()
        date = 'NULL'
        csv_writer.writerow([title, link, date])

File: scrap.js
const request = require('request');
const cheerio = require('cheerio');
const fs = require('fs');
const writeStream = fs.createWriteStream('post.csv');
// Write Headers
writeStream.write(`Title \n`);
request('https://www.npmjs.com/package/cheerio', (error, response, html) => {
  if (!error && response.statusCode == 200) {
    const $ = cheerio.load(html);
    $('li').each((i, el) => {
      const title = $(el)
        .find('code')
        .text()
        .replace(/\s\s+/g, '');
      // Write Row To CSV
      if(title){
        writeStream.write(`${title} \n`);
      }
    });
    console.log('Scraping Done...');
  }
});
